app/locale/nl-NL.coffee

```diff
@@ -480,16 +480,16 @@ module.exports = nativeDescription: "Nederlands (Nederland)", englishDescription
     health: "Gezondheid"
     speed: "Snelheid"
     regeneration: "Herstel"
-#    range: "Range" # As in "attack or visual range"
-#    blocks: "Blocks" # As in "this shield blocks this much damage"
+    range: "Bereik" # As in "attack or visual range"
+    blocks: "Blokkeert" # As in "this shield blocks this much damage"
 #    backstab: "Backstab" # As in "this dagger does this much backstab damage"
     skills: "Vaardigheden"
-#    attack_1: "Deals"
-#    attack_2: "of listed"
-#    attack_3: "weapon damage."
+#   attack_1: "brengt [x damge] toe" # In Dutch we say 'brings [x damge] to...' Alternative: 'veroorzaakt [x damage] (causes [x damage])
+#    attack_2: "of listed" # what is the context?
+    attack_3: "Wapen schade."
```

Comment:
These 3 strings get merged in a sentence like "Deals 120% of listed Warrior weapon damage."